### ISSUE TITLE
Update tabs aria-controls for hidden tabs

### ISSUE THREAD

**Issue by adamduncan** [frend/frend.co#104]
Update spec based on APG outcome: https://github.com/w3c/aria-practices/issues/293

**Comment by mcking65** [comment]
@adamduncan, The ARIA Authoring Practices will stay consistent with the ARIA specification. See my comments in w3c/aria-practices#293 and w3c/aria#531.

I don't think making a change here is a good idea. If there is a change, it should be in user agent requirements or expectations of assistive technologies.

**Comment by adamduncan** [comment]
Thanks for the insights @mcking65! Much appreciated. Closing.
